docs/configuration.md

```diff
@@ -83,6 +83,24 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `WS_CONNECTION_LIMIT_PER_IP`    | "10"     | Maximum amount of connections from a single IP address                                  |
 | `WS_MULTIPLE_ADDRESSES_ENABLED` | "false"  | If enabled eth_subscribe will allow subscription to multiple contract address.          |
 
+## Sample for connecting to Hedera Testnet
+
+```.env
+HEDERA_NETWORK=testnet
+OPERATOR_ID_MAIN=<...redacted...>
+OPERATOR_KEY_MAIN=<...redacted...>
+OPERATOR_ID_ETH_SENDRAWTRANSACTION=
```

Comment:
Drop the `SEND_RAWTRANSACTION` options. They aren't needed if the `MAIN` options are used

Reply:
clarifying: is `..._MAIN` is for ED22519 key-pair and `...ETH_ENDRAWTRANSACTION` is for ECDSA key-pair, and you only need one type of key-pair?

Reply:
You only need `OPERATOR_ID_MAIN` and `OPERATOR_KEY_MAIN` to run the relay.
`..._ETH_SENDRAWTRANSACTION` is legacy logic. We need to investigate to see if we can deprecate/remove it to avoid confusion.
It works it's just you don't need both pairs